---
title="VSD"
output: github_document
title: VisualSARSDiff
---

<!-- README.md is generated from README.Rmd. Please edit that file -->

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>",
  fig.path = "man/figures/README-",
  out.width = "100%",
  title = "VisualSARSDiff"
)
```

# VisualSARSDiff

<!-- badges: start -->
<!-- badges: end -->

The goal of VisualSARSDiff is to sequentially compare different coronaviruses 
spike proteins, an essentialy part of the virus that is known for binding to hosts
and therefore is a main reason in causing infection. With this package, you 
are able to use pairwise sequencing between spike proteins different, already defined
coronaviruses (such as MERS and SARS-CoV-2), as well as compare them in a visual way,
via transmission statistics (worldwide) and other visual sequences (dotplots, etc)

## Installation

You can install the development version of VisualSARSDiff like so:


require("devtools")
devtools::install_github("aryanahmad/VisualSARSDiff", build_vignettes=TRUE)
library("VisualSARSDiff")

##NOTE: Shiny app under construction


## Overview

This is a basic example which shows you how to solve a common problem:


library(VisualSARSDiff)
## I want to compare MERS and SARS-CoV-2
x <- pairwiseSpike("MERS", "COVID-19")
x


## Contributions

All code was created by the author, Aryan Ahmad. 
Genome sequences were used from UNIPROT
pairwiseSpike uses other packages from seqinR and Biostrings

## References

Coghlan, A. (n.d.). Welcome to a little book of R for bioinformatics! — bioinformatics 0.1 documentation. Readthedocs.Io. Retrieved November 22, 2021, from https://a-little-book-of-r-for-bioinformatics.readthedocs.io/en/latest/index.html

Spike glycoprotein. (n.d.-a). Uniprot.Org. Retrieved November 22, 2021, from https://www.uniprot.org/uniprot/K9N5Q8

Spike glycoprotein. (n.d.-b). Uniprot.Org. Retrieved November 22, 2021, from https://www.uniprot.org/uniprot/P0DTC2

Stevens, T. J., & Boucher, W. (2015). Pairwise sequence alignments. In Python Programming for Biology (pp. 208–231). Cambridge University Press.


## Acknowledgements

This package was developed as part of an assessment for 2021
BCB410H: Applied Bioinformatics, University of Toronto, Toronto,
CANADA.
